frontends/mit-open/src/pages/FieldPage/FieldPageSkeleton.tsx

```diff
@@ -61,6 +71,25 @@ const FieldSkeletonProps: React.FC<FieldSkeletonProps> = ({
   const field = useChannelDetail(String(channelType), String(name))
   const urlParams = new URLSearchParams(field.data?.search_filter)
   const displayConfiguration = field.data?.configuration
+  const memoizedUrlParams = useMemo(() => {
+    const urlParams = new URLSearchParams(field.data?.search_filter)
+    return urlParams
+  }, [field.data?.search_filter])
+
+  const urlParamMap: Record<string, string[] | string> = useMemo(() => {
+    return getSearchParamMap(memoizedUrlParams)
+  }, [memoizedUrlParams])
```

Comment:
Minor suggestion: If you're going to memo this, might as well just move the definition of `memoizedUrlParams` into here.